src/odemis/acq/stream/_helper.py

```diff
@@ -1435,11 +1451,13 @@ def _onActive(self, active):
         RepetitionStream._onActive(self, active)
 
 
-class ScannedTemporalSettingsStream(SpectrumSettingsStream):
+class ScannedTemporalSettingsStream(CCDSettingsStream):
```

Comment:
`SpectrumSettingsStream` is a sub-class of `CCDSettingsStream`. The main raison I don't want `ScannedTemporalSettingsStream` to inherit from the former class is for the GUI, as the views are connected to a specific type of stream. Because of this, the chronographs were displayed as spectra. It sort-of worked because the data shape is just 1D anyway, and the spectrum metadata was not found, so the spectrum viewport would show pixels. But it's better that it's picked up by the chronograph viewport, so that the metadata is correctly shown.